cargo-near/src/abi_command/mod.rs

```diff
@@ -0,0 +1,99 @@
+pub mod abi;
+
+#[derive(Debug, Clone, interactive_clap::InteractiveClap)]
+#[interactive_clap(context = near_cli_rs::GlobalContext)]
+#[interactive_clap(skip_default_from_cli)]
+pub struct AbiCommand {
+    /// Include rustdocs in the ABI file
+    #[interactive_clap(long)]
+    pub doc: bool,
+    /// Generate compact (minified) JSON
+    #[interactive_clap(long)]
+    pub compact_abi: bool,
+    /// Copy final artifacts to this directory
+    #[interactive_clap(long)]
+    #[interactive_clap(skip_default_input_arg)]
+    pub out_dir: Option<crate::types::utf8_path_buf::Utf8PathBuf>,
```

Comment:
I don't know the reason behind the use of Utf8PathBuf, so I would love to try and see if PathBuf would work just fine ([we already have it near-cli-rs](https://github.com/near/near-cli-rs/blob/master/src/types/path_buf.rs))

P.S. Obviously, try applying it everywhere we currently have this Utf8PathBuf and if everything works fine, remove the dependency from Cargo.toml as well.